src/drops.cpp

```diff
@@ -100,13 +101,13 @@ int64_t drops::emplace_drops(const name owner, const bool bound, const uint32_t
    }
 
    // generating unbond drops consumes contract RAM bytes to owner
+   const int64_t bytes = amount * get_bytes_per_drop();
    if (bound == false) {
-      const int64_t bytes = amount * get_bytes_per_drop();
-      reduce_ram_bytes(owner, bytes);
-      return bytes;
+      const int64_t newBalance = reduce_ram_bytes(owner, bytes);
+      return {bytes, newBalance};
    }
    // bound drops do not consume contract RAM bytes
-   return 0;
+   return {bytes, 0};
```

Comment:
Should return existing balance instead of 0, seems like this would be a hassle to handle on the UI

Reply:
Done. Added `get_ram_bytes` helper method and then made `generate` return the value of this instead of 0.